sample-viewmodel/src/main/java/com/example/molecule/viewmodel/Presenter.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.molecule.viewmodel
+
+import androidx.compose.runtime.Composable
+import kotlinx.coroutines.flow.Flow
+
+interface Presenter<Event, Model> {
```

Comment:
I don't think people will want to do this and instead probably just want this function to be directly on `MoleculeViewModel`. Top-level, value-returning composables are also testable in isolation the same way instances of this class are.